controllers/wso2is/sfs.go

```diff
@@ -0,0 +1,121 @@
+package wso2is
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/go-logr/logr"
+	wso2v1beta1 "github.com/wso2/k8s-wso2is-operator/api/v1beta1"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	ctrl "sigs.k8s.io/controller-runtime"
+)
+
+func (r *Wso2IsReconciler) defineStatefulSet(m wso2v1beta1.Wso2Is) *appsv1.StatefulSet {
+	containerImage := "rukshanjs/wso2is:v6.1.0"
```

Comment:
This needs to be a configuration

Reply:
Would it be better to include this is a file like configs.go in the root?

Reply:
Added to a /controllers/wso2is/configs.go file